docs/self-hosting.md

```
@@ -98,30 +102,40 @@ gunzip -c tezgah-2024-01-01.sql.gz | \
-`KasapayProvider::new` in `app/server/src/main.rs` in place of `DemoBank`. The
-published image cannot do this for you; it is a starting point for a binary
-you build, not a drop-in payment gateway.
+Set `TEZGAH_PAYMENT_PROVIDER` to `iyzico` or `stripe`, give it the credentials
+that provider needs, and set `TEZGAH_STOCK_LOCATION_ID` — checkout binds and
+the startup log names which bank it is talking to. Both adapters are
+[kasapay](https://github.com/productdevbook/kasapay)'s: tezgah writes no
+payment provider, and `app/server/src/bank.rs` is the whole of what this
+binary does about that — a `match` from a name in the environment onto an
+adapter crate, wrapped so tezgah sees only its own `PaymentProvider` trait.
+
+    TEZGAH_PAYMENT_PROVIDER=iyzico
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)